### ISSUE TITLE
Endponts Security module error for MITRE ATT&CK drill down.

### ISSUE THREAD

**Issue by aliaxonbaig** [wazuh/wazuh#25963]
|Wazuh version: 4.9.0 |Component: Endpoints-Summary|Install type: Docker |Install method: Docker pull |Platform Linux |
|---|---|---|---|---|

When trying to open Top Tactics from End  Point Summary, we encounter the below error:

![image](https://github.com/user-attachments/assets/6cd5382d-50e2-4342-9cf9-ea0f0acad52c)

![image](https://github.com/user-attachments/assets/c12e8924-ba0e-48e6-b1ab-cb119a011460)

![image](https://github.com/user-attachments/assets/5523992a-3866-4cbc-a201-c07e32ea7331)

Logs:

![image](https://github.com/user-attachments/assets/7487a29a-0b90-4201-9b1d-3174b1505783)

**Comment by MouadAssouam** [comment]
yeah me too I find the same problem, did you find any solution for it??
